src/botPage/bot/TradeEngine/Ticks.js

```diff
@@ -58,11 +58,13 @@ export default Engine =>
             );
         }
         getLastDigit() {
-            return new Promise(resolve => this.getLastTick().then(tick => resolve(getLastDigit(tick))));
+            return new Promise(resolve => {
+                this.getLastTick().then(tick => resolve(getLastDigit(tick.toFixed(this.getPipSize()))));
```

Comment:
I think there is a problme with toFixed method and we have to change it. The problme is that when the argument is 2 it functions well but when the argument is 1 for example it rounds the tick digits and then get the last digit. 
Another problem is that toFixed returns a string and if you want to sum up the last digit block with a number, the result would be wrong. Could you plz test these things? I have the same issue and working on them on the other card Thanks